pkg/client/client_venafi_cloud.go

```diff
@@ -0,0 +1,377 @@
+package client
+
+import (
+	"bytes"
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/ed25519"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/json"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/google/uuid"
+	"github.com/hashicorp/go-multierror"
+	"github.com/jetstack/preflight/api"
+	"gopkg.in/yaml.v2"
```

Comment:
Nitpick: We should try and use the latest version of the package, `gopkg.in/yaml.v3`

Reply:
With the move to json, this is no longer required here.